scripts/project-setup-scripts/create-custom-iam-roles-for-projects.sh

```diff
@@ -21,4 +21,5 @@ then
 else
   gcloud iam roles create terra_workspace_can_compute --organization=${ORGANIZATION_ID} --file=terra-workspace-can-compute.yaml
   gcloud iam roles create terra_billing_project_owner --organization=${ORGANIZATION_ID} --file=terra-billing-project-owner.yaml
```

Comment:
these lines will probably error (since the roles already exist) so you can probably comment them out